scripts/imgtool/image.py

```diff
@@ -534,31 +534,38 @@ def pad_to(self, size):
         self.payload += pbytes
 
     @staticmethod
-    def verify(imgfile, key):
+    def verify(imgfile, key, endian="little"):
```

Comment:
I'm not quite sure what happened, but the comments in image.h, and the way we did the TLV, both the image header and the TLV and trailer are all supposed to be little endian. I think we should fix the code to deal with this on big-endian platforms. Either that, or we should change those headers. But, I think I prefer the image header and TLV being the same no matter the endianness of the platform running the code.

Reply:
Yeah, I agree, but `create` is broken then as well, so someone with access to the HW must confirm it is not working. In any case I dropped that commit and will open a new PR only with that commit.

Reply:
Btw, I am quite sure there are big endian users already, so I would assume at least the basics are functional.